orchestrationSpecs/packages/config-processor/sample.yaml

```diff
@@ -0,0 +1,14 @@
+# Sample workflow configuration
```

Comment:
Can this file be removed - or moved elsewhere.  There's already another sampleMigration.yaml file anc we have a limited ability to create a sample from the schema.